packages/block-library/src/navigation/style.scss

```diff
@@ -460,20 +462,19 @@ button.wp-block-navigation-item__content {
 				// Position and style.
 				position: static;
 				border: none;
-				padding-left: 32px;
-				padding-right: 32px;
+				padding-left: 2em;
+				padding-right: 2em;
```

Comment:
Does it make sense to use `em`-based values here? It gets a little weird if you specify a larger text size: 

<img width="286" alt="Screen Shot 2021-12-16 at 9 13 25 AM" src="https://user-images.githubusercontent.com/1202812/146388023-b5991a2b-13ba-4073-9934-26a7dd61110b.png">

Reply:
I guess that change looks a bit slipstreamed in. When I made that change, I actually made it to match up with the change made in #37323. But I'm happy to revisit both of these? What would be ideal in your mind?

Reply:
I think this can be handled separately. Ideally there's some other measurement we can harness for the default (maybe blockGap?). Or else, switching to `rem` would be a little more stable.

Reply:
I'll follow up with rems

Reply:
Followup: https://github.com/WordPress/gutenberg/pull/37478